i18n/en.json

```diff
@@ -3537,4 +3537,4 @@
   "widgets.users_emails_input.valid_email": "Add **{email}**",
   "yourcomputer": "Your computer",
   "youtube_video.notFound": "Video not found"
-}
+}
```

Comment:
Please revert this change. Otherwise CI will fail :wink:

Reply:
Thanks @hanzei 🙂 I've squashed my commits and `rebase`d on `master`, which should remove any changes that I made to this file.